src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx

```diff
@@ -623,12 +645,21 @@ const EditTestCase = (props: EditTestCaseProps) => {
               Changes {action}d successfully but the following{" "}
               {severityOfValidationErrors(validationErrors)}(s) were found
             </h3>
+            {timezoneUpdated && <ul>Timezones updated in json</ul>}
             <ul>{valErrors}</ul>
           </div>
         ) : (
-          `Test case updated successfully with ${severityOfValidationErrors(
-            validationErrors
-          )}s in JSON`
+          <div>
+            <h3>
+              Test case updated successfully with{" "}
+              {severityOfValidationErrors(validationErrors)}s in JSON
+            </h3>
+            {timezoneUpdated && (
+              <ul style={{ listStyle: "inside" }}>
+                <li>Timezones updated in json</li>
```

Comment:
I think there is specific message mentioned in ticket to be displayed when timezone is updated

Reply:
Thanks I forgot to check on that. Updated